pagopa/io-app
# fix: [IOBP-796] Payment method details scroll with new header second level

## Short description
This PR fixes the scrolling issue in the payment method details screen.

## List of changes proposed in this pull request
- Replaced the deprecated `BaseScreenComponent` with the use of custom header second level
- Added header animation when scrolling;

## How to test
- Go into payment home screen
- If you have an already onboarded payment method, please go in the details screen by tapping on it -> otherwise start the payment method onboarding by tapping on the "Add" CTA on the up-right corner;
- Once you're in the payment method details screen, you should be able to scroll the view even if it's bigger than your device screen.

## Preview

|Before|After|
|-|-|
|<video src="https://github.com/user-attachments/assets/92fcb99e-3ac9-431c-8290-c4e0a83519d2" />|<video src="https://github.com/user-attachments/assets/0e4fb9f8-4c48-4fb4-827d-a738b5a3ffc3" />
